cmake/dependencies.cmake

```diff
@@ -0,0 +1,22 @@
+# ReadOSM
+if(DEFINED ReadOSM_DIR)
+  find_package(ReadOSM CONFIG REQUIRED)
```

Comment:
@mwoehlke-kitware  I had to add CONFIG here, or else I would end up in the FIndReadOSM.cmake file
So maybe these were just swapped? (No CONFIG seems to use the Find file...)

Reply:
Yeah, I suspect this was just an oversight...